Project Management Process.md

```diff
@@ -0,0 +1,287 @@
+# IPFS Project Management Process
+
+**Status: work in progress, not even a full draft yet**
+
+## Introduction
+TODO: describe the purpose of this document
+
+- this is an opt-in process, nobody in the community is *required* to use this process
+
+## Table on Contents
+- [TL;DR](#tldr)
+- [Projects](#projects)
+- [Roles](#roles)
+  - [Product Owners](#product-owners)
+  - [Project Leads](#project-leads)
+  - [Project Managers](#project-managers)
+- [Structure](#structure)
+  - [Overview](#overview)
+  - [Goals](#goals)
+  - [Backlog](#backlog)
+  - [Milestones](#milestones)
+  - [Project Roadmap](#project-roadmap)
+  - [Organization Roadmap](#organization-roadmap)
+- [Workflow](#flow)
+  - [Pipelines](#pipelines)
+    - [Stages](#stages)
+    - [Flow](#flow)
+    - [Example](#example)
+  - [Weekly Updates](#weekly-updates)
+  - [Roadmap Updates](#roadmap-updates)
+- [Implementation Guideline](#implementation-guideline)
+
+## TL;DR
+
+The IPFS project management process has the following **cadence**:
+
+Weekly Updates
+- once a week (2 weeks?)
+- discuss last week's progress
+- discuss the plans for next week
```

Comment:
Agreed, I'm missing that part from our weekly and would much rather discuss who's doing what this week instead of going through what happened last week.